src/Resources/ResourceManager/Implementation/Policy/GetAzurePolicyAssignment.cs

```diff
@@ -170,7 +170,7 @@ private bool IsResourceGet(string resourceId)
         /// </summary>
         private string GetResourceId()
         {
-            return this.Id ?? this.MakePolicyAssignmentId(this.Scope, this.Name);
+            return this.Id ?? this.GetPolicyArtifactFullyQualifiedId(this.Scope, Constants.MicrosoftAuthorizationPolicyAssignmentType, this.Name);
```

Comment:
>GetPolicyArtifactFullyQualifiedId [](start = 35, length = 33)

Is this is just a refactoring, or also a bug fix (same question applies to other cmdlets)? Can you now delete MakePolicyAssignmentId(), or is it still in use?

Reply:
It's just refactoring. I forgot to remove it.